README.md

```diff
@@ -216,6 +220,8 @@ add_swagger_documentation \
    base_path: '/super/api'
 ```
 
+`host` and `base_path` are also accepting a `proc` or `lambda`
```

Comment:
In addition to that, I think providing some trivial example would be super helpful in the docs here:

```ruby
add_swagger_documentation(
  base_path: proc { |request| request.host =~ /^api/ ? '/' : '/api' }
)
```